source/crud/crud.md

```diff
@@ -2369,6 +2391,8 @@ aforementioned allowance in the SemVer spec.
 
 ## Changelog
 
+- 2024-08-30: Add sort option to `updateOne`.
```

Comment:
```suggestion
- 2024-08-30: Add sort option to `replaceOne` and `updateOne`.
```

Based on the changes above, I assume this applies to both methods.

On a separate note, does this change also need to be introduced to the new `bulkWrite` spec?